java/client/src/org/openqa/selenium/remote/tracing/EventAttribute.java

```diff
@@ -34,6 +34,22 @@ public static EventAttributeValue setValue(long value) {
     return new EventAttributeValue(value);
   }
 
+  public static EventAttributeValue setValue(String[] value) {
+    return new EventAttributeValue(value);
+  }
+
+  public static EventAttributeValue setValue(Long[] value) {
```

Comment:
Prefer primitive types.

Reply:
OpenTelemetry APIs for array attributes accepts boxes types so stuck with that.